### ISSUE TITLE
write-off with VAT impossible outside a bank statement reconcile

### ISSUE THREAD

**Issue by fclementic2c** [odoo/odoo#68215]
Impacted versions: 13

Steps to reproduce: https://docs.google.com/document/d/1Ggf7Y-KqBzTaDxjDwAy5FE_mU1vY-MfQnmVvNBSz6sU/edit?usp=sharing

Support ticket number submitted via odoo.com/help (optional):

**Comment by ray-odoo** [comment]
It doesn't work to use a Vendor Credit Note?  Don't some Vendors issue a Credit Note to reduce the VAT in the case where you pay early? Also, is there really no VAT on the Bill?

**Comment by fclementic2c** [comment]
No, @ray-odoo. Vendors do not create credit note for financial discount. This is part of the payment agreement.
if you pay within 10 days for ex. then you can pay 98% of the amount.

**Comment by ray-odoo** [comment]
Ok. So YOU can't create a credit (after intitially posting a bill at full price with full VAT) to manage both the payment discount and the VAT discount (you can get a Journal Item for each with whatever account you choose)?
